README.md

```diff
@@ -0,0 +1,67 @@
+
+# Overview
+
+Terraform module for setting up infrastructure for [Shunting Yard](https://github.com/HotelsDotCom/shunting-yard).
+
+For more information please refer to the main [Apiary](https://github.com/ExpediaInc/apiary) project page.
+
+## Variables
+| Name | Description | Type | Default | Required |
```

Comment:
please update this as per `variables.tf`

Reply:
Done.